client/task-list/tasks/payments/RemotePayments/index.js

```diff
@@ -19,27 +18,22 @@ import { sift } from '../../../../utils';
 
 export const RemotePayments = ( { query } ) => {
 	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
-	const { getOption } = useSelect( ( select ) => {
+	const {
+		getOption,
+		getPaymentMethodRecommendations,
+		isFetching,
+	} = useSelect( ( select ) => {
 		return {
 			getOption: select( OPTIONS_STORE_NAME ).getOption,
+			getPaymentMethodRecommendations: select( ONBOARDING_STORE_NAME )
+				.getPaymentMethodRecommendations,
+			isFetching: select( ONBOARDING_STORE_NAME ).hasFinishedResolution(
```

Comment:
This needs to be `! hasFinishedResolution` right?

```suggestion
			isResolving: ! select( ONBOARDING_STORE_NAME ).hasFinishedResolution(
```

Reply:
Thanks for catching this, I opt for `select( ONBOARDING_STORE_NAME ).isResolving` instead!